server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -333,6 +333,9 @@ export function create(
 			const clientIPAddress = request.ip ? request.ip : "";
 			const networkInfo = getNetworkInformationFromIP(clientIPAddress);
 			// Tenant and document
+			Lumberjack.info(
```

Comment:
Can we make this a little less wordy:

Lumberjack.info(
  `Network info for createDoc: 
    clientIP=${clientIPAddress}, 
    isPrivateLink=${networkInfo.isPrivateLink}, 
    enablePrivateLinkNetworkCheck=${enablePrivateLinkNetworkCheck}, 
    linkId=${networkInfo.privateLinkId}`
);